[WIP] Revamped octane tutorial (#1002)
ember-learn/guides-source: guides/release/tutorial/10.md

> We've been refactoring our existing code for a while, so let's implement a new feature for a change. Let's work on the site-wide navigation bar next.

Reviewer: ```suggestion
We've been refactoring our existing code for a while, so let's change gears and implement a new feature: the site-wide navigation bar.
```

      Contact
    </LinkTo>
  </div>
</nav>
```

Next, we will add our `<NavBar>` component to the top of each page:

```handlebars {data-filename="app/components/index.hbs" data-diff="+1"}
<NavBar />
<Jumbo>
  <h2>Welcome to Super Rentals!</h2>
  <!-- ...snip... -->
</Jumbo>
```

```handlebars {data-filename="app/components/about.hbs" data-diff="+1"}
<NavBar />
<Jumbo>
  <h2>About Super Rentals</h2>
  <!-- ...snip... -->
</Jumbo>
```

```handlebars {data-filename="app/components/contact.hbs" data-diff="+1"}
<NavBar />
<Jumbo>
  <h2>Contact Us</h2>
  <!-- ...snip... -->
</Jumbo>
```

Voilà, we made another component!

<div class="cta">
  <div class="cta-note">
    <div class="cta-note-body">
      <div class="cta-note-heading">Zoey says...</div>
      <div class="cta-note-message">
        `<NavBar />` is a shorthand for `<NavBar></NavBar>`. Component tags must always be closed properly, even when you are not passing any content to them, as in this case. Since this is pretty common, Ember provides the alternative self-closing shorthand to save you some typing!
      </div>
    </div>
    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
  </div>
</div>

> Everything looks great in the browser, but as we know, you can never be too sure. So let's write some tests!

Reviewer: ```suggestion
Everything looks great in the browser, but as we know, we can never be too sure. So let's write some tests!
```
